pkg/runner/runner.go

```diff
@@ -145,26 +142,18 @@ func (r *Runner) cleanup(options *flags.Options) (err error) {
 	noDotsRelease := strings.Replace(options.Release, ".", "", 1)
 	imageList, err := r.imgDataTarget.GetVersions(noDotsRelease, options.Channel, options.Arch)
 	if err != nil {
+		log.Infof("Error getting image list")
```

Comment:
Just Info here?
I'm not sure if it's needed, but I saw you changing it somewhere else.